src/KeyValidatorTrait.php

```diff
@@ -23,6 +19,12 @@
      */
     protected function validateKey($key)
     {
+        $matched = preg_match('/^[A-Za-z0-9_.]{1,64}$/', $key);
+
+        if (1 !== $matched) {
+            throw new InvalidArgumentException('Can\'t validate the specified key');
```

Comment:
Should we include the invalid key in the response? Seems like it, so people know what broke.  Unless there's a security risk there?